DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -440,7 +440,11 @@ template <typename T>
 std::ostream& operator<<(std::ostream& os, const BufferDataView<T>& view)
 {
     auto nBytes = std::max(view.desc.sizeInBytes, (uint64_t) view.desc.initialValues.size());
-    uint32_t elementCount = nBytes / Device::GetSizeInBytes(view.desc.initialValuesDataType);
+    uint64_t elementCount = nBytes / Device::GetSizeInBytes(view.desc.initialValuesDataType);
+    if(elementCount > std::numeric_limits<unsigned int>::max())
+    {
+        throw std::invalid_argument("Buffer size is too large");
+    }
```

Comment:
This shouldn't be necessary: Device::GetSizeInBytes returns a uint32_t and you're stored it in a uint64_t.

Reply:
the issue is the for loop is using an uint32_t and span uses size_t in the loop which can change size depending on platform.  Do we need to index larger than uint?